src/Counter.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+interface State {
+  count: number;
+}
+
+export class Counter extends React.Component<{}, State> {
+  state = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState((prevState) => {
```

Comment:
Досить зручно зразу робити деструкторизацію
```suggestion
    this.setState(({ count }) => {
```